oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java

```diff
@@ -56,7 +56,7 @@ public RestResource() {
     @Path("/health-check")
     @Produces(MediaType.APPLICATION_JSON)
     public String healthCheck() {
-        return "{\"status\" : \"running\"}";
+        return new String("{\"status\":\"running\"}").trim();
```

Comment:
This is super silly. Never do this and read about java string pool. Just hit `java string pool` in google and you will see a lot of articles.